use-case-guides/workday-request-time-off/README.md

```diff
@@ -209,52 +178,49 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
 ![Screenshot 2024-07-04 at 8.19.38 PM.png](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Screenshot_2024-07-04_at_8.19.38_PM.png)
 
 
-> ✅ **Inform your Creator Studio developer that they can use your Workday connector to build their use case** 🚀
+## Step 2: Create a Connector in Creator Studio
+Follow the authentication guide to [create a working Workday connector in Creator Studio](https://developer.moveworks.com/creator-studio/resources/authentication-guide/?id=workday#create-a-connector-and-test-in-creator-studio) with the credentials that you received from your Workday administrator.
+
+![Untitled](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Screenshot%202024-07-23%20at%2010.04.23 AM.png)
 
-### **Step 3: Build in Creator Studio**
+## Step 3: Build in Creator Studio
 
-We will be building this use case as a query triggered path along with an iPaaS to chain APIs 3 and 4 together in order to submit a time off request for a user. We will be following the P1: Sequential Action Chaining design pattern in your iPaaS to build this use case
+We will be building this plugin as a Path along with an iPaaS to chain APIs 3 and 4 together in order to submit a time off request for a user.
 
-### Design Pattern P1: Sequential Action Chaining
+### Plugin Architecture Diagram
 
-![Untitled](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Untitled.png)
+![Untitled](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Untitled.png)
```

Comment:
References a `resovler` which does not exist

Reply:
Is AuthZ not a part of API3? I don't think AuthN / AuthZ should get disconnected from the core.

Reply:
![CleanShot 2024-08-13 at 13 25 48@2x](https://github.com/user-attachments/assets/c9c920df-40e5-4ef7-9c90-6c37aaf25f32)
This feels extremely similar to what we do for the PC Mock. Is this really necessary. I would recommend we drop the architecture diagram unless you have a section in the style guide explaining why it should be there.